[pull] primary from flybywiresim:primary (#390)
Valastiri/fbw-userguide: docs/pilots-corner/advanced-guides/flight-guidance/autoland.md

In summary:

- Autoland is not a replacement for learning and training how to land manually.
- Special training and certifications are required for real pilots.
- Aircraft and airport must have the appropriate capability.
- It requires a high amount of preparation and limiting conditions need to be checked constantly (e.g. maximum 
  crosswind).
- Autoland requires high focus from the flight crew monitoring the systems and to always be ready to take over for a 
  go around. Pilot monitoring (PM) usually will look outside the aircraft while pilot flying (PF) will watch the 
  instruments.

## Real World Considerations

### When to Use Autoland

When low visibility operations (LVO) CAT III are enforced use of Autoland is required. LVO operations are either 
announced via ATIS or ATC. 

Performing an Autoland is permissible for CAT II / CAT III runways even when low visibility procedures (LVP) are not 
enforced and this may even extend to CAT I runways. 

Autoland can also be subject to local procedures and authorities.

### When Not to Use Autoland

- ATC restrictions
- Airport capabilities missing or restrictions in place.
- Aircraft capabilities missing and limitations exceeded.

> - If CAT1 is displayed on the FMA, Autoland is not authorized.

Reviewer: ```suggestion
- If CAT I is displayed on the FMA, Autoland is not authorized.
```

Reviewer 2: It's actually showing CAT1 :)

    (checked with ATIS or ATC) the runway **must be considered a CAT I runway**.

    Crews should be alert to ILS beam / signal contamination since the LOC/GS signal strength may not be guaranteed.

CAT I runway considerations before using Autoland:

- Pilots must be ready to disconnect the AP if any ILS fluctuations are detected.
- CAT II should at least be indicated on the FMA and the flight crew should apply the respective CAT II / CAT III 
  FCOM procedures.
- Visual contact should be established at the CAT I Minimum at the latest.

### Aircraft Limitations

All Airbus aircraft are certified to land automatically. However, limitations and conditions specified in the FCOM 
must be taken into account. Be aware that other not-so-obvious Autoland-limitations, such as maximum airfield 
altitude, maximum (minimum) GS angle or maximum runway slope, must also be considered. 

In addition, the flight crew must monitor possible day-to-day technical restrictions (stated in the MEL), or the 
consequence(s) of a failure that may have occurred during the flight and that may downgrade landing capability.

Autoland many not be performed outside of these limitations:

> - Max. Crosswind: 20ft

Reviewer: ```suggestion
- Max. Crosswind: 20kt
```

- Max. Headwind: 30kt
- Max. Tailwind: 10kt
- Glide slope angle smaller than -2.5° or greater than -3.25°
- Airport elevation above 9 200 ft
- Aircraft weight below 44.000 kg (97.004 lb)
- Aircraft weight above the maximum landing weight
- Automatic landing is not allowed below -2.000 ft pressure altitude

Minima: 

- ILS CAT II: MDH 100ft
- ILS CAT III SINGLE: MDH 50ft
- ILS CAT III DUAL: no DH (minimum RVR 75m)

> ## Autoland HowTo

Reviewer: ```suggestion
## Autoland How To
```

See [Beginner Guide Preparation and Checklist for Landing](../../beginner-guide/landing.md#3-preparation-and-checklist-for-landing)

- At 1000ft: callout "one thousand".
- At 500ft: now callouts every hundred feet.
- Shortly after the 400 callout and before 350ft the FMA must show LAND in green to signal that the aircraft is now 
  in landing mode.
- At 350ft: check the ILS course on the PFD.
- At 150ft: The white line signifying the ground on the PFD attitude indicator starts to move up.
- At 100ft: callout "hundred".
- At 50ft: callout "fifty" and now callouts every 10 ft
- At 40ft: the FMA must show FLARE or otherwise the flight crew needs to either perform a manual landing if 
  visual references are acquired or perform a go around.
- Autothrust FMA should switch to THRUST IDLE.
- Between about 20ft and 10ft the aircraft corrects its crosswind crab angle to bring its longitudinal axis in line 
  with the runway.
- At 10ft: callout "Retard" commands the PF to set the throttle levers to idle.
- At touchdown:
    - FMA shows ROLL OUT - autopilot keeps aircraft on center line
    - Both thrust levers to reverse thrust (idle or max as appropriate)

>     - PM checks and calls out: "touchdown", "spoilers", "reversers", "auto brake", "descel".

Reviewer: ```suggestion
    - PM checks and calls out: "touchdown", "spoilers", "reversers", "auto brake", "decel".
```

- FMA not showing `LAND` at 350ft: 
    - Perform a go-around, or
    - A transition to a manual landing may be possible during a CAT II approach and at or above 80ft if visual 
      references are acquired
- [Autoland Warning Light](#autoland-warning-light) at or below 200ft:
    - Perform a go-around
    - A transition to a manual landing may be possible during a CAT II approach and at or above 80ft if visual 
     references are acquired
- FMA not showing `FLARE` at 40ft:
    - Perform a go-around
- FMA not showing `ROLL OUT` at touchdown:
    - Disconnect the autopilot and track center line using visual references or localizer signal down to taxi speed.

If below 1.000ft AAL (above airport level) but higher than 100ft RA (radio altitude):

- Perform a **go-around** in case of:
    - AP loss (cavalry charge)                
    - Amber caution (single chime)            
    - [Autoland Warning Light](#autoland-warning-light) (below 200’ RA)  
    - Triple click (downgrading of capability)
- Proceed in case of:
    - Fire (red warning: Engine or APU)       
    - Smoke (red warning: Avionics, Cargo, Lavatory)

>     - Smokes/Fumes in the cockpit/Cabin

Reviewer: ```suggestion
    - Smokes/Fumes in the Cockpit/Cabin
```
Or cockpit/cabin -- just capilization check
